sqlstore/src/main/java/uk/ac/bbsrc/tgac/miso/persistence/impl/HibernateServiceRecordDao.java

```diff
@@ -95,9 +95,9 @@ public boolean remove(SequencerServiceRecord ssr) throws IOException {
    */
   private void removeAttachments(long recordId) {
     try {
-      for (String filename : misoFilesManager.getFileNames(SequencerServiceRecord.class, String.valueOf(recordId))) {
+      for (String filename : misoFilesManager.getFileNames(ServiceRecord.class, String.valueOf(recordId))) {
         try {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Extract this nested try block into a separate method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1141)